changelog/fragments/align-kb.yaml

```
@@ -110,4 +110,11 @@ entries:
   - description: >
       For Go-based operators, Makefile scaffold was improved
     kind: "change"
-    breaking: false
\ No newline at end of file
+    breaking: false
+  - description: >
+      Fix issue to run Go-based projects on OCP by changing the manager pod template's security context `runAsUser: 65532` to `runAsNonRoot: true`. More info [#kubernetes-sigs/kubebuilder#1978](https://github.com/kubernetes-sigs/kubebuilder/pull/1978).
+    kind: "bugfix"
```

This isn't a bugfix, since the manager runs fine on vanilla k8s.
```suggestion
    kind: "change"
```